cmd/user-ssh-keys-agent/Makefile

```diff
@@ -14,11 +14,25 @@
 
 DOCKER_REPO ?= "quay.io/kubermatic"
 GOOS ?= $(shell go env GOOS)
+GOPATH ?= $(shell go env GOPATH)
+GOCACHE ?= $(shell go env GOCACHE)
 
 .PHONY: build
 build:
-	GOOS=$(GOOS) CGO_ENABLED=0 go build -o ./_build/user-ssh-keys-agent
+	docker run \
+		-e GOCACHE=/go/.cache/go-build \
+ 		-e CGO_ENABLED=0 \
+		-v $(GOPATH)/src/github.com/kubermatic/kubermatic/:/go/src/github.com/kubermatic/kubermatic \
+		-v $(GOCACHE):/go/.cache/go-build \
+		-v $(GOPATH)/src/github.com/kubermatic/kubermatic/cmd/user-ssh-keys-agent/_build:/go/_build \
```

Comment:
Why is it not enough to mount the repo two lines above this one?

Reply:
You are right, gonna remove it.